src/picologging/handlers.py

```diff
@@ -464,3 +464,57 @@ def doRollover(self):
                     addend = 3600
                 now_rollover_at += addend
         self.rollover_at = now_rollover_at
+
+
+class BufferingHandler(picologging.Handler):
+    """
+    A handler class which buffers logging records in memory. Whenever each
+    record is added to the buffer, a check is made to see if the buffer should
+    be flushed. If it should, then flush() is expected to do what's needed.
+    """
+    def __init__(self, capacity):
+        """
+        Initialize the handler with the buffer size.
+        """
+        picologging.Handler.__init__(self)
```

Comment:
please use `super().__init__()`

Reply:
Just to be clear, do we want to be this different from the rest which specifically decided not to use `super().__init__()`?

Reply:
The more people I talk to about this pattern, the more I get told I'm being pedantic. Leave it as-is :-)